# 케라스(Karas)

In [4]:
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

import numpy
import tensorflow as tf

X = numpy.array([0,1,2,3,4,5])
y = X * 2 + 1

model = Sequential()
model.add(Dense(1,input_shape=(1,)))
model.compile('SGD', 'mse')
model.fit(X,y,epochs=1000, verbose=0) #verbose보이게 할지 안보이게 할지

predict = model.predict(X).flatten()

print('Target:', y)
print('Predict:',predict)

Target: [ 1  3  5  7  9 11]
Predict: [ 0.9996033  2.999715   4.999827   6.999939   9.000051  11.000162 ]


* 정답이 1인데 내가 예측한 것은 0.9996033, 
* 정답이 3인데 내가 예측한 것은 2.999715,
* 정답이 5인데 내가 예측한 것은 4.999827,
...